syngenta_digital_alc/apigateway/handler_requirements.py

```diff
@@ -30,8 +32,8 @@ def wrapper(event, context, schema_path=""):
                     )
 
                     logger.log(
-                        level='ERROR',
-                        log=f"{request.path_parameters} has timed out! Sending human readable message to user."
+                        level="ERROR",
+                        log=f"{request.path_parameters} has timed out! Sending human readable message to user.",
```

Comment:
Take this seriously.

Reply:
This is a log message that we see internally. I honestly think it's helpful for us, internally in datadog, to see that people are be notified of timeouts. The user sees the above message, `"This API call timed out. Please try more, smaller queries, or reach out to our help-connect channel!"`

Reply:
It doesn't matter if it is internal or external. Change it.